### ISSUE TITLE
Thursday

### ISSUE THREAD

**Issue by Codingmaestre2015** [WestMEC-Coding-Session1/Week-27-Bell-Work#3]
Good Afternoon,

**Scenario:** You are tired of searching the web for jobs. You have been at it for six weeks with no leads and nothing in the way of gainful employment. Just when things are looking most bleak, and you are ready to try your hand at rodeo clowning, something catches your eye. A new company, requesting a young and ambitious web-development specialist, is looking to hire a full-time employee at a substantial salary. You REJOICE! Finally, a good fit. 

Your first step is to polish your resume. 

*What are the 5 key elements of a good resume and what should you never have on it?* 

You are called by human resources from the company which you have eagerly applying to work at. They inform you that you have an interview tomorrow at 8:00 AM. You are confident in your skills as a programmer and are very ambitious. This company is a perfect fit for you and a lot is riding on you procuring employment there. Plus, you are tired of living with your parents, even though your mom does your laundry. 

*What are you going to do to make a good first impression?*

*How are you going to dress?*

*How will you respond to questions to which you don't know the answer?*

**We will talk about whether or not you get hired, thus being able to leave your parents home, in a bit.**

**Comment by deherrera-marques-wm** [comment]
Professionalism: Email and number and actual name
Consistency in employment: you don’t have to have all the jobs you have done if it is under a year don’t use it.
Complication in resume: Don’t make it long and make sure it is toward the job you want.
Color: Some of the bold text a different color, or graph for what you are good in.
Do not put a picture unless you want a modeling job.
Weird attachment: Use a Pdf attach to email to make it easier to open.
Question: Testing to see if you are capable of doing something so simple, Read every part take your time answering what is asked of you.

I would dress professional, and shake their hands thanking them for letting me come in to talk with them today. If I don't know how to answer a question I would simply ask them if they would rephrase the question they have asked me, and when the interview is done I would try to send one last email thanking them for even considering me.

**Comment by clocksin-nick-wm** [comment]
1. professional email
2. only important or relevant information for the job
3. small amount of job listings
4. no personal photos
5. answer any response questions

I will make a good first impression by dressing and having a good introduction.

I will make a answer that satisfies the interviewer.

**Comment by le-ben-wm** [comment]
The 5 key elements of a good resume are professional contact info, employment consistency, important, relevant information, use a PDF, and read ever part of the job description or ad and answer all questions.
To make a good first impression, give a handshake that matches the other person's handshake strength, seem engaged during the interview, wear clean clothes, dress nice, speak clearly and professionally.
To respond to a question I don't know the answer to, I would answer the question to the best of my ability, or say that I don't know the answer, but am willing to learn about it if that is relevant.

**Comment by schweikert-zachary-wm** [comment]
5 key good things in a resume are a formal contact email, an answer to the question the employer is asking, positive references, previous work experience, and the resume must be specific to the job you are applying for.
To make a good first impression I will try to impress the employer with my knowledge of the job by making a pun with that information. 
I will dress formally and in the correct sized clothing.
I will always let the interviewer know if I do not know the answer to appear honest but also throw something extra to impress the interviewer and distract the interviewer slightly.

**Comment by steele-ariel-wm** [comment]
The five key elements to a resume are: 
1. Having professional contact information. 
2. Leave off side jobs so it looks like you have consistent employment.
3. make it simple
4. no profile picture 
5. no weird attachments.
I will make a good first impression by dressing nice and thank them for taking the time to interview me.
I would dress professionally. 
I'll respond to questions I don't know by either asking them to clarify what they mean or try to think of an answer they'll like.

**Comment by gonzalez-jose-wm** [comment]
The five things to do with a resume are: have a professional looking email, make it recognizable (Stand Out), stability in employment, keep it short and sweet, and lastly use PDF for your resume. 

The first thing I would do is shake the person that is interviewing me hand to show respect and thank him/her for the opportunity for coming. I will be dress with a suit and tie so I can look as professional as possible. The best way to respond to a question that you do not know the question to is saying something in the lines of "I can answer your question partly" then give a response of "please let me know if you have anything you would like me to add". It is ok to ask for a second to think about the question.

**Comment by burton-jacob-wm** [comment]
Do :
1. Make a recognizable résumé.
2. Make sure content is relevant to the job.
3. Be Professional with emails and phone numbers.
4. Read job description completely for other important information.
5. Include only important information

Do Not :
1. Have personal photos.
2. Make a résumé too long or complicated.

First Impressions : 
To make good first impressions, 
1. Start with a great introduction and be confident.
2. Dress to impress.
3. Be honest but use critical thinking skill to the best ability.

**Comment by capp-brendan-wm** [comment]
**5 Key Points**

- Don't have to much information on the resume, be specific and concise about it.
- Don't add a personal photo if it no required.
- Don't add content to the resume no one can access or that is unneeded.
- Don't add in side jobs or short-term jobs.
- Have professionalism on the resume meaning have all the basic required information and nothing extra.

Dress properly for business that shows a proper appearance and greet the potential employer.

If there is a question I do not know how to answer, I would attempt to answer the question to the best of my ability and be honest.

**Comment by mcclure-calvin-wm** [comment]
Professional email, relevant info, don't list too many previous jobs, no pictures, make sure to answer questions

To make a first impression I am going to carry myself in a proud and admirable manner. Try to look like someone worth knowing. Paired with that I will use a better vocabulary than usual to show off my intelligence. I will make sure to use buzzwords and other phrases that will outline my legitimacy. I will  dress for the job and not underdress. If a question ever comes up that I don't know the answer to- fake it till you make it! When that isn't an option try to smoothly talk your way around it, or suggest that it be my concern of learning and say it in a way that pleases the interviewer.

**Comment by brey-matthew-wm** [comment]
The key 5 elements are:
-You have to have an appropriate email
-having relevant information]
-consistency in jobs
-no selfies
-answer a question if they email back

To make a good first impression, I would give the interviewer a firm handshake, and look him in the eyes when he talks. I would smile a decent amount, and be respectful. 

I would dress for success

I would go out on a straw man and talk about something else great about me, jk i would probably respectfully say that I am not sure of the answer.

**Comment by Garcia-Rogelio-WM** [comment]
To polish your resume should have a professional email. It should not have a personal picture. Have consistent jobs. have awards that relate to the job. I would wear something nice. And answer questions honestly exaggerate a little but not too much

**Comment by casey-leevon-wm** [comment]
-No pictures

-Add a hint of color

-Only list jobs that lasted a year or longer

-Add only PDF document forms

-Know the job you are applying for

–––––––––––––––––––––––––––––––––––––––––––––––––––––––––––––––––––––––––––––––––––
I would dress nice, answer to the best of my ability, and maintain my normal reactions.

**Comment by riggs-jacob-wm** [comment]
The five key elements of a good resume are:
Professionalism.
Job consistency.
Simplicity.
No photos.
Completion in questions and resume.

I will make a good first impression by dressing appropriately and introducing myself in a kind and well-mannered way. 
If I did not know the answer to a question I would answer to the best of my ability and try to be as honest as possible.

**Comment by osborn-daucen-wm** [comment]
1. Professionalism, Consistency in employment, Complication in resume, have no picture, and evidence that you read the job description fully to completion.

2. To make a good impression you want to lighten the mood. Show that you are charismatic. However, be professional.

3. Dress for success. Where nice clothes but don't go too overboard. At the very least, a collard shirt or a button-up. Suits may be preferred.

4. Don't lie or give false information to an interviewer. Be honest and answer the question to your fullest ability. There's nothing worse than seeming like you are qualified but don't actually have the skills or knowledge you claimed to have.

**Comment by Keyan-Ngai-wm** [comment]
-Professional email
-the resume is relevant to the job you're applying to
-a short and simple listing of past experiences
-no profile pictures
-references
In order to make a good first impression your appearance is key. Try to keep it professional and wear something that fits and is more formal. Also have a good attitude and be prepared for what may be asked, and be honest during your interview.

**Comment by marquez-mollee-wm** [comment]
5 Key Element:

- profesional email

- consistent employment

- short

- pdf format

- specific for the job

I would dress professional and I would make a good first impression by smiling and acting confident.

I would confidently bs the answers I don't know. Fake it till you make it right.

**Comment by hogencamp-devon-wm** [comment]
1. All of your contact Information
2. Show past employment
3. Keep it simple and Relevant
4. Answer any question from job posting
5. Make it simple to open
6. Add some color to make your resume stand out

- Never have pictures
- No bad email names
-Never more than 1 page

First thing is smile and shake their hand and introduce yourself. Be there early and be very respectful not so full of yourself but very humble. Make sure you don't waste the interviewer's time by rambling.

Make sure you are dressed appropriately and that you have everything you need to show them you would be a good candidate for the job. 

If you don't know the answer to a question explain your method that you would go to find the the answer or solve the problem. With other questions make sure you are keeping it simple and not rambling as well.  Don't be so shy when questions are asked but confidently answer them to the best of your ability.

**Comment by breeah-wm** [comment]
1. Five key-elements are a professional email, relevant information regarding the job, short-term jobs, and to answer any questions in the ad. 
2. To make a good first impression, I would show the interviewer what I know about the job and have a positive and energetic attitude. 
3. I would dress professionally. 
4. I would ask them interviewer to clarify, and if I still have no answer, I'll just tell them I'm not sure.

**Comment by mason-gavin-wm** [comment]
**5 Key elements of a good resume**
_1. Professionalize in header (contact, email, name)
2. Having consistency in employment
3. Need relevance to what you are applying to
4. Little color, make it recognizable 
5. Good action key words_
**5 Never to have on a resume**
_1. No pictures, doesn't need to know psychical appearance
2. No weird text Document
3. Lots of color (pink paper, e.t.c.)
4. Small dull words
5. Not to long_
<ul>
<li>
Look clean and professional.
</li>
<li>
I would dress appropriately for the interview, (ex. tux, tie, slacks, dress shoes).
</li>
<li>
I would answer the best I can if I do not know the question, but try to put it that I will know it and work on it to better myself, or whatever it is.
</li>
</ul>

**Comment by diaz-andy-wm** [comment]
The resume is important and I would hone it with the key details like:
<div>making my contact information look professional, 
make my work history appear stable, 
try to keep it short by using only, if not mostly, 
relevant information, 
use some color or small graphics to be remembered but not too much, 
and use a pdf. 
<div>
I would also make sure not to put a selfie or any awkward or eliminating pieces of information on the resume. At the interview, to make a good first impression I would shake the person's hand and ask them for their name after they ask for mine if they do and remember it so at the end of the interview I can tell them thank you and have a nice day by name. I would wear something really dressed up and professional like a button up shirt, slacks and a tie. Most importantly, I would try to keep calm and confident.

**Comment by pierce-eric-wm** [comment]
SHOULD HAVE
1) Professional Email
2) The job you are applying for
3) Make it simple
4) Some color (but not too much)
5) Jobs you have had(only if you have had it for a long time to show consistency)
DON'T PUT IN ON
Picture of yourself, bad grammar, bad organization

For me to make a good impression i would make myself look nice and be early to the meeting. I would dress in nice attire. If I get asked any questions I don't wish to answer I will still answer them because I value honesty.

**Comment by shephard-azaniyah-wm** [comment]
A good resume stands out appropriately to catch the readers attention. It must also be related to the job that you are applying to. A good resume is organized and has a completed name and email on it. make sure to put full time jobs, but no summer jobs. Keep each resume you submit to the different jobs you are applying for. Another major tip is to read the entire job opening for there might be some questions you need to answer.

I will come in excited and with a smile that will last through the interview.

I will dress of higher class compared to the job standards that I am applying to so they understand that I take my work seriously.

I will try to define the question to my best ability without any need to lie.

**Comment by thorpe-sean-wm** [comment]
Five key elements to a good resume would be that it states specifically what job he/she is going for as well as a professional formating, Proper grammar and punctuation, positive references,  and any previous work experience in which lasted longer than a year. 
Make sure never to add personal pictures and don't make it long winded and bland.
To make a good first impression it is very important to greet them formally, shake their hand and make sure you sit up straight and keep a serious attitude. Remember to dress professionally and answer all questions honestly unless it doesn't appease the interviewer.

**Comment by cunningham-corbin-wm** [comment]
<b>Key Elements</b>:
1. No Picture
2. Make it unique
3. Use formal email and actual name
4. Only information on you that will actually be of interest to the employer
5. Only list major previous jobs (not small one-time things)
<b>How to Make a Good First Impression</b>:
1. Be confident and happy (or at least pretend to be).
2. Get there early, do NOT be late.
3. Make eye contact.
4. Don't appear to be too relaxed or confident.
<b>Dress Code</b>:
1.   MEN: Definitely wear a SUIT. If you don't have one, rent one.
WOMEN: Wear a nice DRESS. If you don't have one, rent one.
2. Do NOT dress casual under any circumstance.
<b>Responding to Questions you don't know the Answer to</b>:
1. Ask for clarification; maybe you just misunderstood them, you want to display your knowledge.
2. If you still don't know, be honest with the person; lying to them won't get you anywhere.
